dGame/EntityManager.h

```diff
@@ -20,6 +20,13 @@ class EntityManager {
 public:
 	void Initialize();
 
+	/**
+	 * Add entity to the entity lookup map
+	 * @param objectId Object ID
+	 * @param entityPtr Entity pointer 
+	*/
+	void AddEntity(const LWOOBJID objectId, Entity* entityPtr) { m_Entities.insert_or_assign(objectId, entityPtr); };
```

Comment:
im not sure how much of a fan I am of exposing this API as it implies anyone and thing can add an entity when in reality every single entity must go through create entity first.